malwarecage/web/src/components/About.js

```diff
@@ -50,20 +49,24 @@ class About extends Component {
                         </div>
                     </View>
                 </div>
-                <div className="container">
-                    <table className="table table-striped table-bordered wrap-table">
-                        <thead>
-                            <tr>
-                                <th>Plugin name</th>
-                                <th>Description</th>
-                                <th>Version</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            {plugins}
-                        </tbody>
-                    </table>
-                </div>
+                {plugins.length ?
+                    <div className="container">
+                        <table className="table table-striped table-bordered wrap-table" >
+                            <thead>
+                                <tr>
+                                    <th>Pluginname</th>
+                                    <th>Description</th>
+                                    <th>Version</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {plugins}
+                            </tbody>
+                        </table>
+                    </div>
+                :
+                    <p>No plugins are installed. Visit our <a href="https://hackmd.io/rcdkXnEISPiPAnwo8INbGw#Integration-guide">documentation</a> to learn about Malwarecgae plugins and how they can be used and installed.</p>
```

Comment:
This link should not be public. Documentation is WIP and will be placed in Wiki or Readthedocs.
```suggestion
                    <p>No plugins are installed. Visit our <a href="https://github.com/CERT-Polska/malwarecage/wiki">documentation</a> to learn about Malwarecage plugins and how they can be used and installed.</p>
```